### DISCUSSION TITLE
Add support for wireguard or vpn

### DISCUSSION THREAD

**Discussion by SamanKia** [dchristl/esp32_nat_router_extended#144] [Ideas]
Thanks for excellent job. It would be great if you add a feature to use wireguard or any other tunneling system over NAT so all of connected clients automatically use that tunnel to connect to specific network. It's gonna be perfect to help people like us in IRAN to use free internet without our government filtering, of course we put our credential of wireguard or other tunneling server account ourselves.

**Comment by dchristl** [comment]
Hello,

this is already adressed #74
